core/src/main/java/gov/nist/secauto/metaschema/core/metapath/item/node/AbstractNodeItemFactory.java

```diff
@@ -249,18 +250,44 @@ private IAssemblyNodeItem getCycledInstance(
    * @return the stream of descendant instances
    */
   @NonNull
-  protected Stream<INamedModelInstanceAbsolute> getNamedModelInstances(@NonNull IContainerModelAbsolute container) {
+  protected Stream<? extends INamedModelInstance> getNamedModelInstances(@NonNull IContainerModelAbsolute container) {
```

Comment:
Non-blocking: sorry to ask conceptual questions and we can take this to Element/Gitter if you'd prefer, but without more code context for me to understand I am struggling to understand: what makes a container of field "absolute" here?

Reply:
I have been strugling with the naming here. There are two types of `INamedModelInstance`: `INamedModelInstanceAbsolute` and `INamedModelInstanceGrouped`. The latter are modeled as part of a choice group, while the former are directly part of the model. An `INamedModelInstance` represents both.

Reply:
OK thank you just wanted to make sure I got it. If I can think of tactical documentation recommendations to help present-tense A.J. and future A.J.'s (plural), I will mull it over for tomorrow morning.

Reply:
If I want document this the classes are diffed and part of the PR for the scope of review, so I will have to revisit this in a separate PR if I want to do this. I'll suppress it for now.